doc/man3/OPENSSL_init_crypto.pod

```diff
@@ -162,7 +162,7 @@ The OPENSSL_cleanup() function deinitialises OpenSSL (both libcrypto
 and libssl). All resources allocated by OpenSSL are freed. Typically there
 should be no need to call this function directly as it is initiated
 automatically on application exit. This is done via the standard C library
-L<atexit(3)> function. In the event that the application will close in a manner
+atexit() function. In the event that the application will close in a manner
```

Comment:
Because we don't link to other C runtime functions (see OPENSSL_malloc.pod for example which doesn't have links to strdup, malloc, and so on)